sencha-workspace/packages/slate-cbl/src/view/student/DashboardController.js

```diff
@@ -0,0 +1,137 @@
+/*jslint browser: true, undef: true *//*global Ext,Slate*/
+Ext.define('Slate.cbl.view.student.DashboardController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.slate-cbl-student-dashboard',
+    requires: [
+        'Slate.cbl.model.Student',
+        'Slate.cbl.model.Competency',
+        'Slate.cbl.view.student.skill.OverviewWindow'
+    ],
+
+    config: {
+        id: 'slate-cbl-teacher-dashboard', // workaround for http://www.sencha.com/forum/showthread.php?290043-5.0.1-destroying-a-view-with-ViewController-attached-disables-listen-..-handlers
```

Comment:
won't be an issue in practice because we won't be loading the teacher dashboard and student dashboard in the same runtime session, but this should be `slate-cbl-student-dashboard`